go/trivial.go

```diff
@@ -3,33 +3,51 @@ package main
 import (
 	"fmt"
 	"os"
+	"strings"
 	"time"
 
-	"github.com/uber/jaeger-client-go"
+	jaeger "github.com/uber/jaeger-client-go"
 	"github.com/uber/jaeger-client-go/config"
 
-	"github.com/opentracing/opentracing-go"
+	opentracing "github.com/opentracing/opentracing-go"
 )
 
+func errExit(errstr string) {
+	fmt.Println(errstr)
+	os.Exit(1)
+}
+
 func main() {
+	var (
+		hostPort string
+	)
+
+	if len(os.Args) == 2 {
+		hostPort = os.Args[1]
+		if !strings.Contains(os.Args[1], ":") {
+			hostPort += ":6831"
+		}
+	}
+
 	// 1) Create a opentracing.Tracer that sends data to Zipkin
-	cfg := config.Configuration{
-		Sampler: &config.SamplerConfig{
-			Type:	"const",
-			Param:	1,
-		},
+	cfg := &config.Configuration{
 		Reporter: &config.ReporterConfig{
-			LogSpans:		true,
-			BufferFlushInterval:	1 * time.Second,
+			LogSpans:            true,
+			BufferFlushInterval: 1 * time.Second,
```

Comment:
this is not needed since `defer closer.Close()` would flush everything

Reply:
I'm pretty sure this was already there?